frontend/src/pages/Admin/Components/Faq/AddFaq/AddFaq.jsx

```diff
@@ -96,7 +96,7 @@ export class AddFaq extends React.Component {
     e.preventDefault();
     if (this.handleValidation()) {
       let tags = this.state.tags;
-      return fetch(`${END_POINT}/faq`, {
+      return fetch(`${END_POINT}/faq/faq`, {
```

Comment:
[question] shouldn't it be `getFaq`?

Reply:
`/faq`  route  for post method to add faq and i just added base` /faq` route  whereas` /getFaq` is for get Method/
```
const faq = require('./post');
const getfaq = require('./getFaq');
router.post('/faq', validation(FAQValidationSchema), faq);
router.get('/getFaq', getfaq);

```

Reply:
base route 
```
const faq = require('./faq');
`router.use('/faq', faq);`
```

Reply:
can we please make it `/postFaq` instead as `faq/faq` looks confusing